tests/e2e/dpa_deployment_suite_test.go

```diff
@@ -465,10 +619,7 @@ var _ = Describe("Configuration testing for DPA Custom Resource", func() {
 			},
 			WantError: false,
 		}, nil),
-	}
-
-	awsTests := []TableEntry{
-		Entry("AWS Without Region No S3ForcePathStyle with BackupImages false should succeed", Label("aws"), InstallCase{
+		Entry("AWS Without Region No S3ForcePathStyle with BackupImages false should succeed", Label("aws", "ibmcloud"), InstallCase{
```

Comment:
TODO test if developers in ibm environment can really run these tests

Reply:
BSL will not be available, because no s3 url is passed, but that not a problem, we check only DPA in these tests